docs/guide/repository.md

```diff
@@ -599,32 +494,26 @@ This cache backend is using [Redis, a in-memory data structure store](http://re
 |`password`|Optional setting if there is a password to connection to a given Redis server in plain text over the network.|
 |`database`|Optional setting to specify a given Redis database to use.|
```

Comment:
setting above is now not relevant anymore

Reply:
Last question (hopefully): do you mean just the `database` setting, or all settings in this table?

Reply:
all

Reply:
well, it's different

If we adapt to state it as ` **Available provider settings**` and adjust naming to the once in the link I gave above, then it can stay in, but it needs to be clarified with examples how they are applied to the provide config.

Reply:
But it's easier to remove the section and copy examples from links above, so in this case provider beolow would be:
```
+services:
    cache.redis:
        parent: cache.adapter.redis
        tags:
            - name: cache.pool
              clearer: cache.app_clearer
              # Examples from vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/RedisTrait.php:
              # redis://localhost:6379
              # redis://secret@example.com:1234/13
              # redis://secret@/var/run/redis.sock/13?persistent_id=4&class=Redis&timeout=3&retry_interval=3
              provider: 'redis://%env(CACHE_DSN)%'
```

Reply:
actually no, remove the setting section, we will need to explain the supported DSN, or better would be to get symfony to do it so we can link to it.

Like this one:
https://symfony.com/doc/3.4/components/cache/adapters/redis_adapter.html
https://symfony.com/doc/3.4/components/cache/adapters/memcached_adapter.html